extensions/resteasy-reactive/quarkus-resteasy-reactive-jaxb-common/runtime/src/main/java/io/quarkus/resteasy/reactive/jaxb/common/runtime/serialisers/JaxbMessageBodyReader.java

```diff
@@ -49,7 +49,8 @@ protected boolean isReadable(MediaType mediaType, Class<?> type) {
         String subtype = mediaType.getSubtype();
         boolean isApplicationMediaType = "application".equals(mediaType.getType());
         return (isApplicationMediaType && "xml".equalsIgnoreCase(subtype) || subtype.endsWith("+xml"))
-                || (mediaType.isWildcardSubtype() && (mediaType.isWildcardType() || isApplicationMediaType));
+                || (mediaType.isWildcardSubtype() && (mediaType.isWildcardType() || isApplicationMediaType))
+                || ("text".equals(mediaType.getType()) && "xml".equalsIgnoreCase(subtype));
```

Comment:
I think `text/xxx+xml` is not supported and it should be. 
This method should be refactored to validate the following media types: application/xml, application/xxx+xml, text/xml, text/xxx+xml.

Reply:
> That was my initial idea too and I initially updated the code to also include those, however after some searching I didn't find any occurrences of `text/xxx+xml` just `text/xml`. For `application/xxx-xml` there are some use case, also see https://www.iana.org/assignments/media-types/media-types.xhtml and https://javaee.github.io/javaee-spec/javadocs/javax/ws/rs/core/MediaType.html
> 
> That's why I ended up creating only a case for `text/xml`

Yeah, we don't have any examples of using suffixes with either `application/xml` or `text/xml` (We do for Json: https://github.com/quarkusio/quarkus/blob/main/extensions/panache/hibernate-reactive-rest-data-panache/deployment/src/test/java/io/quarkus/hibernate/reactive/rest/data/panache/deployment/AbstractGetMethodTest.java#L133).

However, if this is supported for `application/xml`, it should work the same for `text/xml`, just in case users start using it.

Reply:
Thanks!